librz/core/cmd/cmd_print.c

```diff
@@ -6478,3 +5792,692 @@ RZ_IPI RzCmdStatus rz_print_columns_hex_words_handler(RzCore *core, int argc, co
 	int user_rows = argc > 1 ? rz_num_math(core->num, argv[1]) : -1;
 	return bool2status(print_hexdump_columns(core, user_rows, "pxw"));
 }
+
+RZ_IPI RzCmdStatus rz_print_equal_d_handler(RzCore *core, int argc, const char **argv) {
+	int i, min = -1, max = 0, dict = 0, range = 0;
+	bool histogram[256] = { 0 };
```

Comment:
Since it's not the "hot" path and just a command handler, I think it's fine as it is for the sake of code simplicity. Otherwise, we might just use `RzBitvector` there.

Reply:
then is ok.